### PR TITLE
added user section to the api endpoints

### PR DESCRIPTION
**Pull request by MichaelErmer**
Add User:
POST /server/<serverid>/user/add
Formdata: username & password

Del User:
POST /server/<serverid>/user/del/<userid>

### CODE REVIEW

File: app/api.py [hunk 1]
```diff
@@ -224,6 +224,59 @@ def logs(self, id):
         return Response(json.dumps(logs, sort_keys=True, indent=4), mimetype='application/json')
 
     @conditional(auth.login_required, auth_enabled)
+    @route('<id>/user/del/<user>', methods=['POST'])
```
Comment (alfg) [inline app/api.py]: Can we use `<id>/user/<user>` as a `delete

File: app/api.py [hunk 2]
```diff
@@ -224,6 +224,59 @@ def logs(self, id):
...
+    def user_del_user(self, id, user):
+        """ Creates user
```
Comment (alfg) [inline app/api.py]: Should be "Deletes user".

File: app/api.py [hunk 3]
```diff
@@ -224,6 +224,59 @@ def logs(self, id):
...
+        
+        json_data = {
+            "user_id": user,
+            "deleted": 'Success'
+        }
+        return Response(json.dumps(json_data, sort_keys=True, indent=4), mimetype='application/json')
+
+
+    @conditional(auth.login_required, auth_enabled)
+    @route('<id>/user/new', methods=['POST'])
```
Comment (alfg) [inline app/api.py]: A POST on `<id>/user` should be good, instead of `/new` at the end.

**Comment by alfg** [comment]
Hi there, this looks good. I made a couple of comments if you don't mind fixing those. I'll review, test and merge it in tonight if all goes well. 

Thanks so much!

-Alf

**Comment by MichaelErmer** [comment]
done :)

**Comment by alfg** [comment]
Awesome! I'll test it out tonight and merge it in. 

Thanks!

**Comment by alfg** [comment]
Verified that it works, thanks again!

**Pull request merged**